vim.exe.stackdump

```diff
@@ -0,0 +1,14 @@
+Stack trace:
```

Comment:
This file should not be here. You probably have to remove it from git before the .gitignore will work

Reply:
will do